scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ServerHandshaker.java

```diff
@@ -640,7 +640,7 @@ private DTLSFlight receivedClientHello(ClientHello message) throws HandshakeExce
 		if (psk == null) {
 			throw new HandshakeException(
 					String.format("Cannot authenticate client, identity [%s] is unknown", identity),
-					new AlertMessage(AlertLevel.FATAL, AlertDescription.HANDSHAKE_FAILURE, session.getPeer()));
+					new AlertMessage(AlertLevel.FATAL, AlertDescription.UNKNOWN_PSK_IDENTITY, session.getPeer()));
```

Comment:
Wouldn't it make more sense to use `AlertLevel.WARNING` and leave `DTLSConnector.handleExceptionDuringHandshake` untouched?

Reply:
I'm not sure about the "sense". 
It seems, that until now, WARNING is only used for close and therefore the implementation seems not to send other WARNING alert records to the other peers. But I would expect them to be send.
So to use WARNING instead of FATAL seems FMPOV only working in the above case, because the implementation is not as I would expect it.
Therefore I would prefer to stay with FATAL, not that later the sending of WARNING needs to b fixed and the breaks this issue again.

Reply:
@sophokles73 
Do you still want to use a WARNING or could we merge it as it is?

Reply:
LGTM!

Six days waiting for @sophokles73 to explain his comment seems to be enough and we can simply add such changes later.